brigadier/src/main/java/co.aikar.commands/ACFBrigadierManager.java

```diff
@@ -161,4 +149,31 @@
         return root;
     }
 
+    <C> void registerParameters(CommandParameter[] parameters,
```

Comment:
Doesn't need to be a generic method when you're only using RegisteredCommand and the permCheckerSub